Gemfile

```diff
@@ -0,0 +1,10 @@
+source 'https://rubygems.org'
```

Comment:
This should probably match the literal string style for the rest of the project :o